form/src/main/kotlin/com/ritense/form/service/impl/DefaultFormSubmissionService.kt

```diff
@@ -98,33 +98,31 @@ open class DefaultFormSubmissionService(
                 )
             }
 
-            return AuthorizationContext.runWithoutAuthorization {
```

Comment:
@theo-ritense The `CREATE` check already exists but is ignored because of this `runWithoutAuthorization` that wraps everything